src/Tizen.NUI.Wearable/src/public/Title.cs

```diff
@@ -0,0 +1,342 @@
+        /// <since_tizen> 8 </since_tizen>
+        protected override void Dispose(DisposeTypes type)
+        {
+            if (disposed)
+            {
+                return;
+            }
+
+            if (type == DisposeTypes.Explicit)
+            {
+                if (null != leftImage)
+                {
+                    leftImage.Unparent();
+                    leftImage.Dispose();
+                    leftImage = null;
+                }
+                if (null != rightImage)
+                {
+                    rightImage.Unparent();
+                    rightImage.Dispose();
+                    rightImage = null;
+                }
+            }
+            base.Dispose(type);
+        }
+
+        private void UpdateImage()
+        {
+            leftImage.Size.Width = fadeOutWidth;
+            rightImage.Size.Width = fadeOutWidth;
+
+            if (fadeOutWidth > 0)
+            {
+                Color startColor = new Color(fadeOutColor.R, fadeOutColor.G, fadeOutColor.B, fadeOutColor.A * 0.6F);
+                Color endColor = new Color(fadeOutColor.R, fadeOutColor.G, fadeOutColor.B, 0.0F);
+
+                TextureSet leftTextureSet = CreateTexture(startColor, endColor);
+                TextureSet rightTextureSet = CreateTexture(endColor, startColor);
+
+                leftImage.RemoveRenderer(mLeftRenderer);
+                rightImage.RemoveRenderer(mLeftRenderer);
+
+                if (this.EnableAutoScroll)
+                {
+                    mLeftRenderer.SetTextures(leftTextureSet);
+                    mRightRenderer.SetTextures(rightTextureSet);
+
+                    leftImage.AddRenderer(mLeftRenderer);
+                    rightImage.AddRenderer(mRightRenderer);
+                }
+                else if (this.TextDirection == TextDirection.LeftToRight)
+                {
+                    mRightRenderer.SetTextures(rightTextureSet);
+                    rightImage.AddRenderer(mRightRenderer);
+                }
+                else
+                {
+                    mLeftRenderer.SetTextures(leftTextureSet);
+                    leftImage.AddRenderer(mLeftRenderer);
+                }
+            }
+
+            this.Ellipsis = false;
+            this.HorizontalAlignment = HorizontalAlignment.Center;
+            this.VerticalAlignment = VerticalAlignment.Center;
+        }
+
+        private void Initialize()
+        {
+            if (null == leftImage)
+            {
+                leftImage = new ImageView();
+                leftImage.PositionUsesPivotPoint = true;
+                leftImage.ParentOrigin = Tizen.NUI.ParentOrigin.TopLeft;
+                leftImage.PivotPoint = Tizen.NUI.PivotPoint.TopLeft;
+                leftImage.WidthResizePolicy = ResizePolicyType.FillToParent;
+                leftImage.HeightResizePolicy = ResizePolicyType.FillToParent;
+                this.Add(leftImage);
+            }
+            if (null == rightImage)
```

Comment:
Could you explain any case that these inner items are existing before initializing?

Reply:
There is not. This is an unnecessary check code!